README.md

```diff
@@ -18,7 +18,10 @@
 - Restart Home Assistant to install all dependencies
 
 ### Adding or enabling integration
+## My Home Assistant (2021.3+)
```

Comment:
Could you please fix the title hierarchy so that it renders correctly?

I would suggest `####` for this and Manual.

```suggestion
#### My Home Assistant (2021.3+)
```